### ISSUE TITLE
Support for OpenAI Whisper transcription from CLI

### ISSUE THREAD

**Issue by HamzaETTH** [tkarabela/pysubs2#73]
I have some Whisper JAX (https://huggingface.co/spaces/sanchit-gandhi/whisper-jax) transcription in .txt files and I'd like to use the CLI tool to convert them to .srt . Could that be added ?

**Comment by HamzaETTH** [comment]
I thought I included a sample, my mistake. Thank you.

```
[01:14.500 -> 01:15.500]  Okay.
[01:15.500 -> 01:17.500]  You know you can't smoke weed and drive, right?
[01:17.500 -> 01:19.500]  That's a DWI, man.
[01:19.500 -> 01:20.500]  Really?
[01:20.500 -> 01:21.500]  Yeah.
[01:22.500 -> 01:25.000]  And the reason I'm saying that is I can smell it, right?
[01:25.000 -> 01:29.000]  And I don't know if it's literally all over you.
[01:29.000 -> 01:32.000]  Or what's all that all over your shirt and your pants?
[01:32.000 -> 01:34.000]  Yeah, I know what I should say.
[01:34.000 -> 01:36.000]  It's like it's all over you, man.
[01:36.000 -> 01:38.000]  Hey, come on, dude.
[01:38.000 -> 01:40.000]  Yeah.
[01:40.000 -> 01:44.000]  So, if you drink and drive, it's a DWI, right?
[01:44.000 -> 01:47.880]  If you smoke weed and drive drive it's the DWI. They're both legal
```

**Comment by tkarabela** [comment]
Implemented in 1.8.0.
